src/Functions.Worker.Extensions.OpenAI/Embeddings/InputType.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.OpenAI.Embeddings;
+
+/// <summary>
+/// Options for interpreting input binding data.
+/// </summary>
+public enum InputType
+{
+    /// <summary>
+    /// The input data is raw text.
+    /// </summary>
+    RawText,
+
+    /// <summary>
+    /// The input data is a file path that contains the text.
+    /// </summary>
+    FilePath,
+
+    /// <summary>
+    /// The input data is a URL that can be invoked to get the text.
+    /// </summary>
+    URL,
```

Comment:
Did we actually implement these? If not, I suggest we remove them and add them back when we do have support for them.

Reply:
We have the implementation in WebJobs for this. Is that what you meant?